azure-devops/azext_devops/dev/team/_format.py

```diff
@@ -4,6 +4,32 @@
 # --------------------------------------------------------------------------------------------
 
 from collections import OrderedDict
+from azext_devops.dev.common.format import trim_for_display, date_time_to_only_date
+
+
+def transform_extension_table_output(result):
+    table_output = [_transform_extension_row(result)]
+    return table_output
+
+
+def transform_extensions_table_output(result):
+    table_output = []
+    for item in sorted(result, key=_get_extension_key):
+        table_output.append(_transform_extension_row(item))
+    return table_output
+
+
+def _transform_extension_row(row):
+    table_row = OrderedDict()
+    table_row['Publisher Id'] = trim_for_display(row['publisherId'], 10)
+    table_row['Extension Id'] = trim_for_display(row['extensionId'], 20)
+    table_row['Name'] = trim_for_display(row['extensionName'], 20)
+    table_row['Version '] = trim_for_display(row['version'], 20)
+    table_row['Last Updated '] = date_time_to_only_date(row['lastPublished'])
+    table_row['States'] = trim_for_display(row['installState']['flags'], 20)
```

Comment:
Suggest: 
Can do more intelligent trimming like add number of flags e.g. 'disabled +2 other'.
Atleast that is how I was planning to do for some pipelines/environment commands.

Open to suggestions even for my own case.

Reply:
kind of border line in this case..
even with this info user will have to switch to json view to get the exact string. This will help only when count is a useful information for the user
and I don't think count is a useful information at least in this case..
(this is more of a case to case thing)